Guides/Migrating.md

```diff
@@ -40,7 +40,15 @@ Also update dependencies version to latest if required.
 
 And run pod install. If successful, running and testing will work correctly. 
 
-#### 9. In example project, add the newly added destination by setting its relative path.
+#### 9. For tests to compile successfully from command line, code signing for the test targets must be set. (It's enough that it's a personal team, Apple ID does not need to have purchased developer program)
```

Comment:
How do we set code signing for the test targets? I am not familiar with this?

Reply:
Select Target and Go to General, and then Choose Development team.
<img width="886" alt="Screen Shot 2020-01-16 at 2 41 44 AM" src="https://user-images.githubusercontent.com/4341880/72542729-4e189700-388d-11ea-98a4-062d54f516ae.png">
 select Test Target.